### PR TITLE
feat: timetable service test

### PR DESCRIPTION
**Pull request by SuperMarcomen**

### CODE REVIEW

File: server/src/test/java/de/uftos/services/TimetableServiceTests.java [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+  @Test
+  void nonExistingTimetableById() {
+    assertThrows(ResponseStatusException.class, () -> timetableService.getById("321"));
+  }
+
+  @Test
+  void createTimetable() {
+    TimetableRequestDto timetable = new TimetableRequestDto("2024");
+    assertDoesNotThrow(() -> timetableService.create(timetable));
```
Comment (danieldietzler) [inline server/src/test/java/de/uftos/services/TimetableServiceTests.java:224]: You should also expect that the repo method has been called

File: server/src/test/java/de/uftos/services/TimetableServiceTests.java [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  }
+
+  @Test
+  void timetableById() {
```
Comment (danieldietzler) [inline server/src/test/java/de/uftos/services/TimetableServiceTests.java:228]: `getTimetableById`

Comment (SuperMarcomen) [reply server/src/test/java/de/uftos/services/TimetableServiceTests.java:103]: done

File: server/src/test/java/de/uftos/services/TimetableServiceTests.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class TimetableServiceTests {
+
+  @Mock
+  private TimetableRepository timetableRepository;
+  @Mock
+  private UcdlRepository ucdlRepository;
+
+  @Mock
+  private ServerRepository serverRepository;
```
Comment (jdiester) [inline server/src/test/java/de/uftos/services/TimetableServiceTests.java:41]: Are the server and ucdl repositories needed anywhere?

**Review by jdiester — APPROVED** [review]

File: server/src/test/java/de/uftos/services/TimetableServiceTests.java (hunk 3 above)
Comment (SuperMarcomen) [reply server/src/test/java/de/uftos/services/TimetableServiceTests.java]: You are right, they are no longer used. I removed them

**Pull request merged**